alvr/server_core/src/connection.rs

```diff
@@ -685,40 +705,42 @@ fn connection_pipeline(
     };
 
     #[cfg_attr(target_os = "linux", allow(unused_variables))]
-    let game_audio_sample_rate =
-        if let Switch::Enabled(game_audio_config) = &initial_settings.audio.game_audio {
-            #[cfg(not(target_os = "linux"))]
-            {
-                let game_audio_device =
-                    AudioDevice::new_output(game_audio_config.device.as_ref()).to_con()?;
-                if let Switch::Enabled(microphone_config) = &initial_settings.audio.microphone {
-                    let (sink, source) =
-                        AudioDevice::new_virtual_microphone_pair(microphone_config.devices.clone())
-                            .to_con()?;
-                    if matches!(
-                        microphone_config.devices,
-                        alvr_session::MicrophoneDevicesConfig::VBCable
-                    ) {
-                        // VoiceMeeter and Custom devices may have arbitrary internal routing.
-                        // Therefore, we cannot detect the loopback issue without knowing the routing.
-                        if alvr_audio::is_same_device(&game_audio_device, &sink)
-                            || alvr_audio::is_same_device(&game_audio_device, &source)
-                        {
-                            con_bail!("Game audio and microphone cannot point to the same device!");
-                        }
+    let game_audio_sample_rate = if let Switch::Enabled(game_audio_config) =
+        &initial_settings.audio.game_audio
+    {
+        #[cfg(not(target_os = "linux"))]
```

Comment:
cargo fmt?

Reply:
Yeah I'm a bit confused. To remove the unused import link for `AudioDevice`, I moved the `use` locally, and somehow that made the whole if block of code indent. in any case that's the rustfmt rules I guess.

Reply:
Ah that's what you were trying to do, because you ended up with a bunch of alvr_audio::AudioDevice's instead

Reply:
sorry, yes. I did not move the use locally, I qualified the struct with the path. I meant to do this how I wrote the code here, no mistakes.

Reply:
it all depends on the context. Here we have many separate cfg blocks, so it's better to use qualified structs. in case we have a larger cfg with multiple instances, then better to use `use`.